plugins/org.eclipse.che.vscode-redhat.java/0.43.0/meta.yaml

```diff
@@ -0,0 +1,14 @@
+id: org.eclipse.che.vscode-redhat.java
+version: 0.43.0
+type: VS Code extension
+name: Language Support for Java(TM)
+title: Language Support for Java(TM) by Red Hat
+description: Java Linting, Intellisense, formatting, refactoring, Maven/Gradle support and more...
+url: https://github.com/redhat-developer/vscode-java/releases/download/v0.43.0/redhat.java-0.43.0.vsix
```

Comment:
Readme on the repo says stable releases are here: http://download.jboss.org/jbosstools/static/jdt.ls/stable/, not the github release page. Please talk to Fred Bricon about what the right location is.

Reply:
it is a good point - need to compare download time from JBoss mirror vs. GH release. I bet we should use the faster mirror even if it will not be consistent with the rest of extensions